npc/re/merchants/eden_market.txt

```diff
@@ -97,7 +96,21 @@ OnClock0000:
 	npcshopupdate "para_hei10",23016,0,1;
 	npcshopupdate "para_key10",7026,10;
 	npcshopupdate "para_key10",7027,10;
-	npcshopupdate "para_wp00",1181,0,1;
+	npcshopupdate "para_wp00",1181,0,1;	// 2h sword - Tae Goo Lyeon
+	npcshopupdate "para_wp00",1231,0,1;	// MATK dagger - Bazerald 
+	npcshopupdate "para_wp00",13039,0,1;	// ATK dagger - Ivory Knife
+	npcshopupdate "para_wp00",1140,0,1;	// 1h sword - Byeollungum
+	npcshopupdate "para_wp00",1142,0,1;	// 1h spear - Hunting Spear
```

Comment:
This is currently [Jeweled Sword](https://www.divine-pride.net/database/item/1142), should it be [Hunting Spear](https://www.divine-pride.net/database/item/1422/hunting-spear-1)?